### PR TITLE
Feature/create accordion component

### PR DESCRIPTION
**Pull request by mosherc**
Wow videos in pull requests, cool

https://user-images.githubusercontent.com/7862535/103022442-f4e3c980-4519-11eb-87ce-f124c34846fd.mov

### CODE REVIEW

File: src/molecules/Accordion/AccordionGroup.style.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    border: none;
+    margin-bottom: ${pm.md};
+    color: ${defaults.linkColor};
+    text-align: left;
+    background: transparent;
+    display: flex;
+    align-items: center;
+    font-size: ${fontSizes.body};
+    transition: color 0.3s, box-shadow 0.3s, background 0.3s;
+    border-radius: 2px;
```
Comment (jennyvb143) [inline src/molecules/Accordion/AccordionGroup.style.ts:25]: Should this be the global default 3px?

File: src/molecules/Accordion/AccordionGroup.test.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            </AccordionHeader>
+            <AccordionDetails>Accordion {accordion}</AccordionDetails>
+          </AccordionItem>
+        ))}
+      </AccordionGroup>
+    );
+  });
+
+  it('displays the accordions correctly', () => {
+    // console.log(wrapper.find('AccordionItem').debug());
```
Comment (jennyvb143) [inline src/molecules/Accordion/AccordionGroup.test.tsx:35]: Remove all `console.log` statements?

File: src/molecules/Accordion/AccordionGroup.test.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+              </AccordionHeader>
+              <AccordionDetails>Accordion {accordion}</AccordionDetails>
+            </AccordionItem>
+          ))}
+        </AccordionGroup>
+      );
+    });
+
+    it('shows the icon', () => {
+      console.log(wrapper.debug());
```
Comment (jennyvb143) [inline src/molecules/Accordion/AccordionGroup.test.tsx:152]: Should this one be removed also?

**Comment by jennyvb143** [comment]
One general issue: The screen reader doesn't jump into the accordion content if you start by clicking into the `Canvas`. I can only get it to read the content if I click an accordion heading first or go backward from `Story`.

File: src/molecules/Accordion/AccordionGroup.style.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { darken, transparentize } from 'polished';
 import { Accordion, AccordionItemPanel } from 'react-accessible-accordion';
 
 import { defaults, colors, pm, fontSizes } from 'stylesheets/global-styles';
-import { Icon } from 'atoms';
+import { Icon } from 'atoms/Icon';
```
Comment (mosherc) [inline src/molecules/Accordion/AccordionGroup.style.ts:6]: https://github.com/styled-components/styled-components/issues/1449#issuecomment-360331968

**Comment by mosherc** [comment]
> One general issue: The screen reader doesn't jump into the accordion content if you start by clicking into the `Canvas`. I can only get it to read the content if I click an accordion heading first or go backward from `Story`.

Not sure I understand exactly what you are running into, but in general it looks like it's working fine for me. As far as reading the content (panel/details) when expanded, I do not think that is required for accessibility. https://www.w3.org/TR/wai-aria-practices-1.1/#accordion and this example https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html do not read the content of the panel when opened. And you can't tab into it unless there are inputs. If you try the demo for the library I used, it works the same way https://react-accessible-accordion.springload.co.nz

**Pull request merged**